### DISCUSSION TITLE
Have popup tabs/splits be presistent

### DISCUSSION THREAD

**Discussion by 7JEO** [Chatterino/chatterino2#3252] [Ideas] [answered]
I like to always my channel visible no matter whos channel im in and having to popup, resize, and position the new window everytime is annoying.

**Comment by IS2511** [comment]
I think what they mean is they want popups to be persistent between restarts.
+1, would like an option (checkbox?) to enable restart-persistent popups, so when I open Chatterino it opens all the popups that were open when I closed it.

**Comment by Mm2PL** [comment]
Technically we have this functionality, we just close other windows for some reason

**Comment by Nerixyz** [comment, marked as answer]
Should be fixed with https://github.com/Chatterino/chatterino2/pull/5081.
